### ISSUE TITLE
EPiServer.Forms: Unable to add more than one form element.

### ISSUE THREAD

**Issue by kjetilmk** [episerver/netcore-preview#116]
AlloyMvcTemplates
EPiServer.Forms 5.0.0-pre-020047
EPiServer.CMS 12.0.1

Stacktrace: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at EPiServer.Forms.EditView.Controllers.FormsElementStore.GetElementTypeId(String typeName)
   at lambda_method2521(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()

**Comment by lunchin** [comment]
Added to internal tracker

**Comment by lunchin** [comment]
I noticed you are using older version.  Can you try the latest in the feed 5.0.0-pre-020194

**Comment by kjetilmk** [comment]
I tried, but there seem to be some dependency trouble?

NU1603: EPiServer.AddOns.Helpers 1.0.0-pre-020194 depends on EPiServer.CMS.AspNetCore (>= 12.0.2-inte-020714 && < 13.0.0) but EPiServer.CMS.AspNetCore 12.0.2-inte-020714 was not found. An approximate best match of EPiServer.CMS.AspNetCore 12.0.2-pre-020757 was resolved.
NU1603: EPiServer.Forms.Core 5.0.0-pre-020194 depends on EPiServer.CMS.AspNetCore.HtmlHelpers (>= 12.0.2-inte-020714 && < 13.0.0) but EPiServer.CMS.AspNetCore.HtmlHelpers 12.0.2-inte-020714 was not found. An approximate best match of EPiServer.CMS.AspNetCore.HtmlHelpers 12.0.2-pre-020757 was resolved.
NU1603: EPiServer.CMS.TinyMce 3.0.0-pre-020127 depends on EPiServer.CMS.UI (>= 12.0.0-inte-021414 && < 13.0.0) but EPiServer.CMS.UI 12.0.0-inte-021414 was not found. An approximate best match of EPiServer.CMS.UI 12.0.0-inte-021419 was resolved.
NU1603: EPiServer.Forms.Core 5.0.0-pre-020194 depends on EPiServer.CMS.UI.Core (>= 12.0.1-pre-022182 && < 13.0.0) but EPiServer.CMS.UI.Core 12.0.1-pre-022182 was not found. An approximate best match of EPiServer.CMS.UI.Core 12.0.1 was resolved.
NU1107: Version conflict detected for EPiServer.Framework. Install/reference EPiServer.Framework 12.0.2-pre-020757 directly to project AlloyMvcTemplates to resolve this issue. 
 AlloyMvcTemplates -> EPiServer.Forms 5.0.0-pre-020194 -> EPiServer.Forms.UI 5.0.0-pre-020194 -> EPiServer.Forms.Core 5.0.0-pre-020194 -> EPiServer.CMS.AspNetCore.HtmlHelpers 12.0.2-pre-020757 -> EPiServer.CMS.AspNetCore.Mvc 12.0.2-pre-020757 -> EPiServer.CMS.AspNetCore.Routing 12.0.2-pre-020757 -> EPiServer.CMS.AspNetCore.Templating 12.0.2-pre-020757 -> EPiServer.CMS.AspNetCore 12.0.2-pre-020757 -> EPiServer.Framework.AspNetCore 12.0.2-pre-020757 -> EPiServer.Framework (= 12.0.2-pre-020757) 
 AlloyMvcTemplates -> EPiServer.CMS 12.0.1 -> EPiServer.Hosting 12.0.1 -> EPiServer.Framework (= 12.0.1).
Package restore failed. Rolling back package changes for 'AlloyMvcTemplates'.
